Lib/defconQt/glyphCollectionView.py

```diff
@@ -375,16 +404,18 @@ def mouseMoveEvent(self, event):
         if event.buttons() & Qt.LeftButton:
             index = self._findEventIndex(event)
             if self._maybeDragPosition is not None:
-                if ((event.pos() - self._maybeDragPosition).manhattanLength() \
-                    < QApplication.startDragDistance()): return
+                if ((event.pos() - self._maybeDragPosition).manhattanLength()
+                        < QApplication.startDragDistance()):
+                        return
                 # TODO: needs ordering or not?
-                glyphList = " ".join(glyph.name for glyph in self.getSelectedGlyphs())
+                glyphList = " ".join(
+                    glyph.name for glyph in self.getSelectedGlyphs())
                 drag = QDrag(self)
                 mimeData = QMimeData()
                 mimeData.setText(glyphList)
                 drag.setMimeData(mimeData)
 
-                dropAction = drag.exec_()
+                # dropAction = drag.exec_()  # unused
```

Comment:
drag.exec_() needs to stay in, only the result code is not used and so dropAction variable can be removed.